src/Auth.ts

```diff
@@ -133,6 +135,10 @@ export class Auth {
           case AuthType.Certificate:
             getTokenPromise = this.ensureAccessTokenWithCertificate.bind(this);
             break;
+          case AuthType.WindowsCertificate:
```

Comment:
Rather than introducing a whole new auth type, I'd suggest we extend the existing `AuthType.Certificate`. That way, we don't need to duplicate the whole logic related to authenticating using the certificate and just need to differentiate in the piece of how we obtain the cert. We can extend the `login` command with a new option named `--certificateFromWindowsCertStore [certificateFromWindowsCertStore]` which should accept a value to lookup the certificate (can we look it up by its name/some other user-readable value or does it need to be thumbprint?). Once we add this, we should also update the docs.

Reply:
will integrate the method into the main Auth.ts

Reply:
Integrated the method into the main Auth class.